python312Packages.weasyprint: 62.3 -> 63.0

phaer opened the pull request:
Weasyprint got a new release.

* Updated ffi.py patch
* updated tinycss2 dependency
* added tinyhtml5 dependency

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [X] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

TomaSajt committed:
sourcegit: init at 2025.1

Co-Authored-By: Peter Kling <1018801+pitkling@users.noreply.github.com>

phaer committed:
python312Packages.tinyhtml5: init at 2.0.0

phaer committed:
python312Packages.weasyprint: 62.3 -> 63.0

nixos-discourse commented:
This pull request has been mentioned on **NixOS Discourse**. There might be relevant details there:

https://discourse.nixos.org/t/prs-ready-for-review/3032/4828

SuperSandro2000 commented on pkgs/development/python-modules/weasyprint/default.nix:
```diff
@@ -90,8 +98,6 @@ buildPythonPackage rec {
     export HOME=$TMPDIR
   '';
 
-  pythonImportsCheck = [ "weasyprint" ];
```
It is not necessary to move this around

SuperSandro2000 commented on pkgs/development/python-modules/weasyprint/default.nix:
```diff
@@ -44,12 +45,13 @@ buildPythonPackage rec {
       pango = "${pango.out}/lib/libpango-1.0${stdenv.hostPlatform.extensions.sharedLibrary}";
       pangocairo = "${pango.out}/lib/libpangocairo-1.0${stdenv.hostPlatform.extensions.sharedLibrary}";
       harfbuzz = "${harfbuzz.out}/lib/libharfbuzz${stdenv.hostPlatform.extensions.sharedLibrary}";
+      harfbuzz_subset = "${harfbuzz.out}/lib/libharfbuzz-subset${stdenv.hostPlatform.extensions.sharedLibrary}";
```
For all of those we shoudl use lib.getLib

phaer replied:
I'd prefer not to block updates with unrelated improvements - as that aspect didn't change with this PR. You are very welcome to update them in a separate one :)

SuperSandro2000 commented on pkgs/development/python-modules/tinyhtml5/default.nix:
```diff
@@ -0,0 +1,33 @@
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  meta = with lib; {
+    description = "A tiny HTML5 parser ";
```
```suggestion
    description = "Tiny HTML5 parser";
```

SuperSandro2000 commented on pkgs/development/python-modules/tinycss2/default.nix:
```diff
@@ -10,32 +10,23 @@
   src = fetchFromGitHub {
     owner = "kozea";
     repo = "tinycss2";
 
```
```suggestion
```

SuperSandro2000 commented on pkgs/development/python-modules/tinycss2/default.nix:
```diff
@@ -10,32 +10,23 @@
 
-    # Tag v1.3.0 is missing the actual version number bump.
-    rev = "bda62b101530588718d931d61bcc343a628b9af9";
+    rev = "v1.4.0";
```
```suggestion
    rev = "v${version}";
```

SuperSandro2000 commented on pkgs/development/python-modules/tinyhtml5/default.nix:
```diff
@@ -0,0 +1,33 @@
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  meta = with lib; {
+    description = "A tiny HTML5 parser";
```
```suggestion
    description = "Tiny HTML5 parser";
```

phaer replied:
This is a literal copy of upstreams github repo description. I prefer not to editorialize them. 

If if I was looking to nit-pick on this, maybe the description in `pyproject.toml` would be a better choice.

SuperSandro2000 replied:
That is written in the contributing guide and we had several treewide PRs already fixing this, so it is not out of my nitpicking.

phaer replied:
It ended up in staging with the indefinite article included, after this very important issue had blocked this PR for two months FYI:

https://github.com/NixOS/nixpkgs/blob/staging/pkgs/development/python-modules/tinyhtml5/default.nix

SuperSandro2000 reviewed (commented):
otherwise lgtm

Moraxyc committed:
alist: add updateScript

r-ryantm committed:
nixt: 2.5.1 -> 2.6.0

booxter committed:
python312Packages.llama-cpp-python: enable for Darwin

For Darwin, there's no CUDA, hence run CUDA tests only for Linux.

Also, if CUDA is used, we need gcc13 or lower, otherwise the build fails
with:

unsupported GNU version! gcc versions later than 13 are not supported!

This change also updates passthru tests so that they are defined on
linux only, since darwin has no CUDA anyway.

FliegendeWurst committed:
oxipng: re-enable tests

FliegendeWurst committed:
nixos/calibre-web: restore compatibility with old dataDir values

r-ryantm committed:
terraform-providers.aiven: 4.31.1 -> 4.33.0

r-ryantm committed:
terraform-providers.migadu: 2024.12.26 -> 2025.1.16

r-ryantm committed:
terraform-providers.opentelekomcloud: 1.36.28 -> 1.36.29

r-ryantm committed:
terraform-providers.datadog: 3.50.0 -> 3.52.1

r-ryantm committed:
plumed: 2.9.2 -> 2.9.3

dotlambda committed:
python313Packages.imap-tools: 1.9.0 -> 1.9.1

Diff: https://github.com/ikvk/imap_tools/compare/refs/tags/v1.9.0...v1.9.1

Changelog: https://github.com/ikvk/imap_tools/blob/v1.9.1/docs/release_notes.rst

Commit:
screen: 4.9.1 -> 5.0.0

Rational of swithing to version v5.0.0.
4.9.1 was broken cross-compilation with GCC 14.

Signed-off-by: Tanel Dettenborn <tanel@ssrc.tii.ae>

nikolaizombie1 committed:
waytrogen: 0.5.8 -> 0.6.3

Wallpaper restore and kill fixes and external script support.

nikolaizombie1 committed:
maintainers: add nikolaizombie1

Commit:
gdlauncher-carbon: 2.0.20 -> 2.0.22

PerchunPak committed:
vimPlugins: move non-github plugins to separate files

The future version won't support anything except GitHub because it
creates a lot of maintenance and it is just easier to handle such
plugins independently.

LordGrimmauld committed:
SDL_ttf: mark insecure

LordGrimmauld committed:
appimageTools: drop SDL1 from fhs environment

alyssais committed:
kanidm_1_3,kanidm_1_4: useFetchCargoVendor

Cargo 1.84.0 seems to have changed the output format of cargo vendor
again, once again invalidating fetchCargoTarball FOD hashes.  It's
time to fix this once and for all, switching across the board to
fetchCargoVendor, which is not dependent on cargo vendor's output
format.

r-ryantm committed:
adguardhome: 0.107.55 -> 0.107.56

r-ryantm committed:
git-cliff: 2.7.0 -> 2.8.0

uninsane committed:
networkmanager-l2tp: fix cross compilation

uninsane committed:
networkmanager-sstp: fix cross compilation

uninsane committed:
networkmanager-openvpn: fix cross compilation

uninsane committed:
networkmanager-openconnect: fix cross compilation

alyssais committed:
timelens: remove

3 years since the last upstream commit.

r-ryantm committed:
toybox: 0.8.11 -> 0.8.12

r-ryantm committed:
cpuinfo: 0-unstable-2024-12-09 -> 0-unstable-2025-01-10

flandweber committed:
sedutil: add meta.mainProgram

wolfgangwalther committed:
oh-my-fish: fix omf-install script

Passing placeholder to substituteAll or replaceVars doesn't work as
expected, it will use the substituteAll's out path instead of the
"caller".

wolfgangwalther committed:
various: replace substituteAll with replaceVars

A few more interesting cases, separated from the big treewide for better
visibility and testing.

uninsane committed:
mutter: fix cross compilation

custom glib-compile-schemas postInstall is no longer required,
as upstream ships its own install script which performs this step.

uninsane committed:
vpnc: support cross compilation

uninsane committed:
networkmanager-vpnc: fix cross compilation

uninsane committed:
gnome-bluetooth_1_0: fix cross compilation

uninsane committed:
blueberry: use `python3.buildPythonApplication` instead of manual wrapping

this fixes the cross-compiled builds to properly use the host python instead of the build python

uninsane committed:
blueberry: use substituteInPlace --replace-fail instead of deprecated --replace

r-ryantm committed:
terraform-providers.baiducloud: 1.21.10 -> 1.21.12

r-ryantm committed:
terraform-providers.okta: 4.12.0 -> 4.13.1

r-ryantm committed:
tpm2-openssl: 1.2.0 -> 1.3.0

r-ryantm committed:
chirpstack-rest-api: 4.10.2 -> 4.11.0

bddvlpr committed:
python3Packages.elevenlabs: 1.9.0 -> 1.50.5

r-ryantm committed:
terraform-providers.buildkite: 1.15.2 -> 1.15.5

jbotwell committed:
ddclient: 3.11.2 -> 4.0.0

The pre-4.0 version is inconsistent with at least one API (Porkbun)
[Release notes](https://github.com/ddclient/ddclient/releases/tag/v4.0.0)

PhilippWoelfel committed:
home-assistant-custom-components.versatile_thermostat: init at 7.1.5

PhilippWoelfel committed:
home-assistant-custom-lovelace-modules.versatile-thermostat-ui-card: init at 1.1.2

alyssais committed:
cargo-web: remove

6 years since the last upstream commit.  Lockfile is too old for
fetchCargoVendor.

alyssais committed:
nextcloud-notify_push: useFetchCargoVendor

Cargo 1.84.0 seems to have changed the output format of cargo vendor
again, once again invalidating fetchCargoTarball FOD hashes.  It's
time to fix this once and for all, switching across the board to
fetchCargoVendor, which is not dependent on cargo vendor's output
format.

ForgottenBeast committed:
vanguards: init at 0.3.1

add derivation for vanguards, a set of scripts that increase security
for tor hidden services by protecting against guard discovery attacks

resources:
https://github.com/mikeperry-tor/vanguards
https://spec.torproject.org/vanguards-spec/index.html?highlight=vanguards

r-ryantm committed:
nixpkgs-hammering: unstable-2024-03-25 -> 0-unstable-2024-12-22

r-ryantm committed:
astal.io: 0-unstable-2025-01-13 -> 0-unstable-2025-01-23

r-ryantm committed:
terraform-providers.tencentcloud: 1.81.152 -> 1.81.164

booxter committed:
python312Packages.llama-cpp-python: fix darwin build w/o Metal devices

booxter committed:
python312Packages.llama-cpp-python: respect NIX_BUILD_CORES

baloo committed:
rpm: fixup %_var macro

Fixes #377078

r-ryantm committed:
waveterm: 0.10.4 -> 0.11.0

r-ryantm committed:
jenkins: 2.479.2 -> 2.479.3

pluiedev committed:
novelwriter: 2.5.3 -> 2.6

r-ryantm committed:
glamoroustoolkit: 1.1.9 -> 1.1.11

r-ryantm committed:
regal: 0.30.0 -> 0.30.2

Tochiaha committed:
python313Packages.compliance-trestle: init at 3.7.0

r-ryantm committed:
terraform-providers.tfe: 0.62.0 -> 0.63.0

r-ryantm committed:
flutter_rust_bridge_codegen: 2.7.0 -> 2.7.1

GaetanLepage committed:
python312Packages.jaxtyping: 0.2.36 -> 0.2.37

Diff: https://github.com/google/jaxtyping/compare/refs/tags/v0.2.36...v0.2.37

Changelog: https://github.com/patrick-kidger/jaxtyping/releases/tag/v0.2.37

GaetanLepage committed:
python312Packages.wadler-lindig: init at 0.1.3

r-ryantm committed:
mailpit: 1.21.8 -> 1.22.0

GaetanLepage committed:
python312Packages.stable-baselines3: 2.4.1-unstable-2025-01-07 -> 2.5.0

Diff: https://github.com/DLR-RM/stable-baselines3/compare/b7c64a1aa4dd2fd3efed96e7a9ddb4d1f5c96112...v2.5.0

Changelog: https://github.com/DLR-RM/stable-baselines3/releases/tag/v2.5.0

Moraxyc committed:
alist: 3.41.0 -> 3.42.0

liberodark committed:
element-{desktop,web}: 1.11.89 -> 1.11.91

techknowlogick committed:
scaleway-cli: 2.35.0 -> 2.36.0

Diff: https://github.com/scaleway/scaleway-cli/compare/v2.35.0...v2.36.0

r-ryantm committed:
thunderbird-bin-unwrapped: 128.6.0esr -> 128.6.1esr

mweinelt committed:
mallard-ducktype: init at 1.0.2

dotlambda committed:
immich: 1.125.3 -> 1.125.6

Diff: https://github.com/immich-app/immich/compare/refs/tags/v1.125.3...v1.125.6

Changelog: https://github.com/immich-app/immich/releases/tag/v1.125.5
           https://github.com/immich-app/immich/releases/tag/v1.125.6

OPNA2608 committed:
lomiri.lomiri-indicator-network: Disable parallel testing

Involves >1 test spinning up & talking to D-Bus, likely cause of current Hydra failure:
https://hydra.nixos.org/build/287207255

r-ryantm committed:
discordo: 0-unstable-2025-01-12 -> 0-unstable-2025-01-27

r-ryantm committed:
terraform-providers.spotinst: 1.207.0 -> 1.208.0

liberodark committed:
openrgb: move to by-name

r-ryantm committed:
python312Packages.es-client: 8.15.2 -> 8.17.1

r-ryantm committed:
terraform-providers.sumologic: 3.0.0 -> 3.0.1

imnotpoz committed:
vimPlugins.repolink-nvim: init at 2023-12-08

dotlambda committed:
paperless-ngx: 2.14.5 -> 2.14.6

Diff: https://github.com/paperless-ngx/paperless-ngx/compare/refs/tags/v2.14.5...v2.14.6

Changelog: https://github.com/paperless-ngx/paperless-ngx/releases/tag/v2.14.6

r-ryantm committed:
svgbob: 0.7.4 -> 0.7.6

r-ryantm committed:
shairport-sync: 4.3.5 -> 4.3.6

r-ryantm committed:
gnmic: 0.39.1 -> 0.40.0

r-ryantm committed:
kittycad-kcl-lsp: 0.1.65 -> 0.1.66

r-ryantm committed:
imgproxy: 3.27.1 -> 3.27.2

r-ryantm committed:
sudo-font: 2.1 -> 2.2

r-ryantm committed:
tui-journal: 0.13.1 -> 0.14.0

r-ryantm committed:
terraform-providers.bigip: 1.22.6 -> 1.22.7

r-ryantm committed:
pay-respects: 0.6.11 -> 0.6.12

r-ryantm committed:
poptracker: 0.29.0 -> 0.30.1

r-ryantm committed:
ddns-go: 6.8.0 -> 6.8.1

r-ryantm committed:
automatic-timezoned: 2.0.45 -> 2.0.50

r-ryantm committed:
sentry-native: 0.7.18 -> 0.7.19

r-ryantm committed:
step-kms-plugin: 0.11.8 -> 0.12.0

edwtjo committed:
slurm: make nvml configurable

r-ryantm committed:
saga: 9.7.1 -> 9.7.2

GaetanLepage committed:
python312Packages.jupyter-server: fix on python3.13 by disabling warning

markuskowa committed:
gromacs: 2024.4 -> 2024.5

r-ryantm committed:
python312Packages.pyiskra: 0.1.14 -> 0.1.15

r-ryantm committed:
python312Packages.scrap-engine: 1.4.1 -> 1.4.2

alyssais committed:
nextcloud-notify_push.test_client: useFetchCargoVendor

Cargo 1.84.0 seems to have changed the output format of cargo vendor
again, once again invalidating fetchCargoTarball FOD hashes.  It's
time to fix this once and for all, switching across the board to
fetchCargoVendor, which is not dependent on cargo vendor's output
format.

GaetanLepage committed:
bacon: 3.9.0 -> 3.9.1

Diff: https://github.com/Canop/bacon/compare/refs/tags/v3.9.0...v3.9.1

Changelog: https://github.com/Canop/bacon/blob/v3.9.1/CHANGELOG.md

luzpaz committed:
vim: migrate homepage URL to https:// schema

r-ryantm committed:
immich-public-proxy: 1.6.2 -> 1.6.3

alyssais committed:
trunk-ng: useFetchCargoVendor

Cargo 1.84.0 seems to have changed the output format of cargo vendor
again, once again invalidating fetchCargoTarball FOD hashes.  It's
time to fix this once and for all, switching across the board to
fetchCargoVendor, which is not dependent on cargo vendor's output
format.

We've checked that fetchCargoVendor produces the same hash on
Darwin (with case-insensitive filesystem) and Linux.

GaetanLepage committed:
vimPlugins.LuaSnip-snippets-nvim: init at 2022-03-17

mweinelt committed:
mallard-ducktype: init at 1.0.2 (#377443)

baloo committed:
rpm: fixup %_var macro (#377194)

r-ryantm committed:
go-migrate: 4.18.1 -> 4.18.2

GaetanLepage committed:
vimPlugins.repolink-nvim: init at 2023-12-08 (#377451)

taku0 committed:
thunderbird-bin-unwrapped: 128.6.0esr -> 128.6.1esr (#377377)

GaetanLepage committed:
vimPlugins.LuaSnip-snippets-nvim: init at 2022-03-17 (#377544)

alyssais committed:
kanata: useFetchCargoVendor

Cargo 1.84.0 seems to have changed the output format of cargo vendor
again, once again invalidating fetchCargoTarball FOD hashes.  It's
time to fix this once and for all, switching across the board to
fetchCargoVendor, which is not dependent on cargo vendor's output
format.

Being optimistic that fetchCargoVendor will have resolved the
case-insensitivity issues.

r-ryantm committed:
darklua: 0.15.0 -> 0.16.0

alyssais committed:
polaris: useFetchCargoVendor

Cargo 1.84.0 seems to have changed the output format of cargo vendor
again, once again invalidating fetchCargoTarball FOD hashes.  It's
time to fix this once and for all, switching across the board to
fetchCargoVendor, which is not dependent on cargo vendor's output
format.

We've checked that fetchCargoVendor produces the same hash on
Darwin (with case-insensitive filesystem) and Linux.

adamcstephens committed:
shairport-sync: 4.3.5 -> 4.3.6 (#377457)

pbsds committed:
polaris: useFetchCargoVendor (#377553)

pbsds committed:
python312Packages.jaxtyping: 0.2.36 -> 0.2.37 (#377307)

imincik committed:
saga: 9.7.1 -> 9.7.2 (#377505)

FliegendeWurst committed:
gdlauncher-carbon: 2.0.20 -> 2.0.22 (#376335)

r-ryantm committed:
minijinja: 2.6.0 -> 2.7.0

GaetanLepage committed:
bacon: 3.9.0 -> 3.9.1 (#377529)

GaetanLepage committed:
python312Packages.jupyter-server: disable failing test on x86_64-darwin

r-ryantm committed:
oauth2c: 1.17.1 -> 1.17.2

FliegendeWurst committed:
nixos/calibre-web: restore compatibility with old dataDir values (#375539)

FliegendeWurst committed:
python3Packages.elevenlabs: 1.9.0 -> 1.50.5 (#376952)

r-ryantm committed:
nimlangserver: 1.8.0 -> 1.8.1

JohnRTitor committed:
team-list: add johnrtitor to android team

dotlambda committed:
nextcloud-notify_push: useFetchCargoVendor (#377087)

dotlambda committed:
python312Packages.pyiskra: 0.1.14 -> 0.1.15 (#377523)

misuzu committed:
networkmanager-vpnc: fix cross compilation (#376860)

misuzu committed:
networkmanager-{l2tp,openconnect,openvpn,sstp}: fix cross compilation (#376687)

markuskowa committed:
gromacs: 2024.4 -> 2024.5 (#377519)

r-ryantm committed:
python312Packages.flux-led: 1.1.0 -> 1.1.3

jtojnar committed:
Merge mutter: fix cross compilation (#376815)

SuperSandro2000 committed:
python312Packages.llama-cpp-python: enable for Darwin (#374698)

SuperSandro2000 committed:
python312Packages.stable-baselines3: 2.4.1-unstable-2025-01-07 -> 2.5.0 (#377341)

misuzu committed:
blueberry: fix cross compilation (#376884)

jian-lin committed:
alist: add update.nix (#358335)

jian-lin committed:
alist: 3.41.0 -> 3.42.0 (#377357)

FliegendeWurst committed:
tesseract: remove explicit phases (#368545)

misuzu committed:
adguardhome: 0.107.55 -> 0.107.56 (#376527)

misuzu committed:
cpuinfo: 0-unstable-2024-12-09 -> 0-unstable-2025-01-10 (#376803)

misuzu committed:
plumed: 2.9.2 -> 2.9.3 (#375577)

misuzu committed:
tpm2-openssl: 1.2.0 -> 1.3.0 (#376907)

misuzu committed:
git-cliff: 2.7.0 -> 2.8.0 (#376558)

misuzu committed:
chirpstack-rest-api: 4.10.2 -> 4.11.0 (#376931)

misuzu committed:
emacsPackages.el-easydraw: 1.2.0-unstable-2025-01-06 -> 1.2.0-unstable-2025-01-16 (#375087)

misuzu committed:
astal.io: 0-unstable-2025-01-13 -> 0-unstable-2025-01-23 (#377124)

misuzu committed:
jenkins: 2.479.2 -> 2.479.3 (#377202)

misuzu committed:
glamoroustoolkit: 1.1.9 -> 1.1.11 (#377270)

misuzu committed:
regal: 0.30.0 -> 0.30.2 (#377272)

misuzu committed:
flutter_rust_bridge_codegen: 2.7.0 -> 2.7.1 (#377304)

misuzu committed:
mailpit: 1.21.8 -> 1.22.0 (#377324)

r-ryantm committed:
python312Packages.pcbnewtransition: 0.4.2 -> 0.5.0

devurandom committed:
netbird: disable shell completion generation when cross compiling (#377572)

Prevent build error when cross compiling:
```
Running phase: installPhase
@nix { "action": "setPhase", "phase": "installPhase" }
/nix/store/gi58xbi7md9sb5gshvnjdkv9n0b3znrw-stdenv-linux/setup: line 228: /nix/store/rpmnk1a3n9ks4d5fj0sr5fkp9nfzdcca-netbird-riscv64-unknown-linux-gnu-0.35.2/bin/netbird: cannot execute binary file: Exec format error
/nix/store/gi58xbi7md9sb5gshvnjdkv9n0b3znrw-stdenv-linux/setup: line 228: /nix/store/rpmnk1a3n9ks4d5fj0sr5fkp9nfzdcca-netbird-riscv64-unknown-linux-gnu-0.35.2/bin/netbird: cannot execute binary file: Exec format error
/nix/store/gi58xbi7md9sb5gshvnjdkv9n0b3znrw-stdenv-linux/setup: line 228: /nix/store/rpmnk1a3n9ks4d5fj0sr5fkp9nfzdcca-netbird-riscv64-unknown-linux-gnu-0.35.2/bin/netbird: cannot execute binary file: Exec format error
installShellCompletion: installed shell completion file `/nix/store/rpmnk1a3n9ks4d5fj0sr5fkp9nfzdcca-netbird-riscv64-unknown-linux-gnu-0.35.2/share/bash-completion/completions/netbird.bash' does not exist or has zero size
```

Inspired by https://github.com/NixOS/nixpkgs/pull/186436.

r-ryantm committed:
python312Packages.bindep: 2.11.0 -> 2.12.0

SuperSandro2000 committed:
paperless-ngx: disable flaky test

nixpkgs-merge-bot[bot] committed:
nimlangserver: 1.8.0 -> 1.8.1 (#377567)

Co-authored-by: daylinmorgan <daylinmorgan@users.noreply.github.com>

GaetanLepage committed:
openrgb: move to by-name (#377416)

r-ryantm committed:
python312Packages.pcffont: 0.0.15 -> 0.0.16

JohnRTitor committed:
team-list: create Hyprland team (#377573)

Resolves https://github.com/NixOS/nixpkgs/issues/377569

Signed-off-by: John Titor <50095635+JohnRTitor@users.noreply.github.com>

dywedir committed:
oxipng: re-enable tests (#375349)

niklaskorz committed:
zed-editor: remove explicit stripDebugList

wegank committed:
nixpkgs-hammering: unstable-2024-03-25 -> 0-unstable-2024-12-22 (#377114)

JohnRTitor committed:
treewide: adopt hypr ecosytem by Hyprland team

Preserved maintainer entries that are not in Hyprland teams.

Removed @wozeparrot as per https://github.com/NixOS/nixpkgs/issues/377569#issuecomment-2619445027

Signed-off-by: John Titor <50095635+JohnRTitor@users.noreply.github.com>

wegank committed:
nixt: 2.5.1 -> 2.6.0 (#372382)

misuzu committed:
gradle: fix cross builds (#371477)

wegank committed:
cargo-web: remove (#377080)

OPNA2608 committed:
lomiri.lomiri-indicator-network: Disable parallel testing (#377414)

SuperSandro2000 committed:
python313Packages.compliance-trestle: init at 3.7.0 (#377275)

SuperSandro2000 committed:
python312Packages.bindep: 2.11.0 -> 2.12.0 (#377581)

leona-ya committed:
element-desktop: 1.11.89 -> 1.11.91 (#377353)

GaetanLepage committed:
python312Packages.jupyter-server: fix on python3.13 by disabling warning (#377509)

dotlambda committed:
immich: 1.125.3 -> 1.125.6 (#377427)

Nobody-alias-N committed:
vital: add missing runtime dependency (#377598)

dotlambda committed:
python313Packages.imap-tools: 1.9.0 -> 1.9.1, paperless-ngx: 2.14.5 -> 2.14.6 (#375682)

FliegendeWurst committed:
vanguards: init at 0.3.1 (#377105)

TomaSajt committed:
sourcegit: init at 2025.1 (#303865)

khaneliman committed:
hyprland-qt-support: init at 0.1.0

khaneliman committed:
hyprpolkitagent: add missing hyprland-qt-support dependency

alyssais committed:
waytrogen: use fetchCargoVendor

Cargo 1.84.0 seems to have changed the output format of cargo vendor
again, once again invalidating fetchCargoTarball FOD hashes.  It's
time to fix this once and for all, switching across the board to
fetchCargoVendor, which is not dependent on cargo vendor's output
format.

khaneliman committed:
hyprsysteminfo: init at 0.1.3

donovanglover committed:
pay-respects: 0.6.11 -> 0.6.12 (#377485)

donovanglover committed:
scaleway-cli: 2.35.0 -> 2.36.0 (#377381)

donovanglover committed:
novelwriter: 2.5.3 -> 2.6 (#377241)

donovanglover committed:
waveterm: 0.10.4 -> 0.11.0 (#377196)

donovanglover committed:
poptracker: 0.29.0 -> 0.30.1 (#377486)

donovanglover committed:
sedutil: add meta.mainProgram (#376780)

khaneliman committed:
xdg-desktop-portal-hyprland: add debug flag

RossComputerGuy committed:
team-list: add johnrtitor to android team (#377571)

wolfgangwalther committed:
various: fix some substituteAll / replace with replaceVars (#376792)

khaneliman committed:
hyprland-qt-support: init at 0.1.0; hyprpolkitagent: add missing hyprland-qt-support dependency; hyprsysteminfo: init at 0.1.3 (#377603)

NickCao committed:
waytrogen: use fetchCargoVendor (#377604)

NickCao committed:
python312Packages.pcffont: 0.0.15 -> 0.0.16 (#377587)

NickCao committed:
python312Packages.pcbnewtransition: 0.4.2 -> 0.5.0 (#377580)

NickCao committed:
python312Packages.flux-led: 1.1.0 -> 1.1.3 (#377576)

NickCao committed:
oauth2c: 1.17.1 -> 1.17.2 (#377562)

NickCao committed:
minijinja: 2.6.0 -> 2.7.0 (#377561)

NickCao committed:
darklua: 0.15.0 -> 0.16.0 (#377552)

NickCao committed:
go-migrate: 4.18.1 -> 4.18.2 (#377548)

NickCao committed:
immich-public-proxy: 1.6.2 -> 1.6.3 (#377540)

NickCao committed:
trunk-ng: useFetchCargoVendor (#377538)

NickCao committed:
step-kms-plugin: 0.11.8 -> 0.12.0 (#377492)

NickCao committed:
sentry-native: 0.7.18 -> 0.7.19 (#377490)

kirillrdy committed:
python312Packages.llama-cpp-python: respect NIX_BUILD_CORES (#377188)

NickCao committed:
automatic-timezoned: 2.0.45 -> 2.0.50 (#377488)

NickCao committed:
ddns-go: 6.8.0 -> 6.8.1 (#377487)

NickCao committed:
terraform-providers.bigip: 1.22.6 -> 1.22.7 (#377479)

NickCao committed:
tui-journal: 0.13.1 -> 0.14.0 (#377475)

NickCao committed:
sudo-font: 2.1 -> 2.2 (#377473)

NickCao committed:
imgproxy: 3.27.1 -> 3.27.2 (#377471)

NickCao committed:
kittycad-kcl-lsp: 0.1.65 -> 0.1.66 (#377463)

NickCao committed:
gnmic: 0.39.1 -> 0.40.0 (#377460)

NickCao committed:
svgbob: 0.7.4 -> 0.7.6 (#377456)

NickCao committed:
terraform-providers.sumologic: 3.0.0 -> 3.0.1 (#377447)

NickCao committed:
python312Packages.es-client: 8.15.2 -> 8.17.1 (#377442)

khaneliman committed:
xdg-desktop-portal-hyprland: add debug flag (#377613)

NickCao committed:
terraform-providers.spotinst: 1.207.0 -> 1.208.0 (#377419)

NickCao committed:
discordo: 0-unstable-2025-01-12 -> 0-unstable-2025-01-27 (#377418)

NickCao committed:
terraform-providers.tfe: 0.62.0 -> 0.63.0 (#377288)

NickCao committed:
terraform-providers.okta: 4.12.0 -> 4.13.1 (#376891)

NickCao committed:
terraform-providers.baiducloud: 1.21.10 -> 1.21.12 (#376890)

NickCao committed:
terraform-providers.datadog: 3.50.0 -> 3.52.1 (#375249)

NickCao committed:
terraform-providers.opentelekomcloud: 1.36.28 -> 1.36.29 (#375248)

NickCao committed:
terraform-providers.aiven: 4.31.1 -> 4.33.0 (#375244)

NickCao committed:
terraform-providers.buildkite: 1.15.2 -> 1.15.5 (#373367)

NickCao committed:
terraform-providers.scaleway: 2.48.0 -> 2.49.0 (#373295)

NickCao committed:
terraform-providers.gridscale: 1.27.0 -> 2.0.2 (#372340)

NickCao committed:
terraform-providers.migadu: 2024.12.26 -> 2025.1.16 (#375247)

NickCao committed:
terraform-providers.tencentcloud: 1.81.152 -> 1.81.164 (#370175)

mweinelt committed:
home-assistant-custom-components.versatile_thermostat: init at 7.1.5, home-assistant-custom-lovelace-modules.versatile-thermostat-ui-card: init at 1.1.2 (#377092)

khaneliman committed:
vimPlugins: move non-github plugins to separate files (#376366)

FliegendeWurst committed:
flex-launcher: init at 2.2 (#371212)

FliegendeWurst committed:
nanobench: init at 4.3.11 (#374269)

mweinelt committed:
SDL_ttf: mark vulnerable; appimageTools: drop SDL1 (#376376)

FliegendeWurst committed:
mp4fpsmod: init at 0.27 (#333636)

FliegendeWurst committed:
toybox: 0.8.11 -> 0.8.12 (#376688)

FliegendeWurst committed:
timelens: remove (#376675)

misuzu committed:
screen: mark as broken on darwin

Disabling PAM doesn't help much.

FliegendeWurst committed:
lldap-cli: 0-unstable-2024-02-24 -> 0-unstable-2024-11-11 (#374905)

dasJ committed:
vim: migrate homepage URL to https:// schema (#377537)

Mic92 committed:
screen: fix darwin build

Mic92 committed:
screen: remove unused configureFlags

misuzu committed:
screen: 4.9.1 -> 5.0.0 (#375953)

GaetanLepage committed:
zed-editor: remove explicit stripDebugList (#377583)
